src/contracts/core/AllocationManager.sol

```diff
@@ -230,6 +239,7 @@ contract AllocationManager is
         RegisterParams calldata params
     ) external onlyWhenNotPaused(PAUSED_OPERATOR_SET_REGISTRATION_AND_DEREGISTRATION) checkCanCall(operator) {
         // Check that the operator exists
+        require(isAVS(params.avs), InvalidAVS());
```

Comment:
Use modifier to be consistent

Reply:
I actually removed this check we validate the opSet exists